src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -0,0 +1,151 @@
+package seedu.addressbook.commands;
```

Comment:
Why is this file here?

Reply:
huh? It's here in the original also right?